Shared/Scripts/BuildXLLkgVersionPublic.cmd

```diff
@@ -1,4 +1,5 @@
-﻿REM @echo off
+REM @echo off
+REM TODO: update LKG_NAME to win-x64 once we start publishing that nuget to MyGet

```

Comment:
This has too wait until we start publishing Microsoft.BuildXL.win-x64 to MyGet.

The pipelines have been updated, just need to wait until we start publishing